lib/network/modules/components/Edge.js

```diff
@@ -1,30 +1,37 @@
 var util = require('../../../util');
 
 var Label = require('./shared/Label').default;
+var ComponentUtil = require('./shared/ComponentUtil').default;
 var CubicBezierEdge = require('./edges/CubicBezierEdge').default;
 var BezierEdgeDynamic = require('./edges/BezierEdgeDynamic').default;
 var BezierEdgeStatic = require('./edges/BezierEdgeStatic').default;
 var StraightEdge = require('./edges/StraightEdge').default;
 
 /**
- * A edge connects two nodes
- * @param {Object} properties     Object with options. Must contain
- *                                At least options from and to.
- *                                Available options: from (number),
- *                                to (number), label (string, color (string),
- *                                width (number), style (string),
- *                                length (number), title (string)
- * @param {Network} network       A Network object, used to find and edge to
- *                                nodes.
- * @param {Object} constants      An object with default values for
- *                                example for the color
- * @class Edge
+ * An edge connects two nodes and has a specific direction.
+ *
+ * @class
```

Comment:
Yes, for the following reasons:

- Not one of the documented `@param` values exist; they appear to be an artifact of previous changes.
- The `@param` definitions are in the wrong place anyway; they should be defined at the constructor, not the class.

A new correct header comment has been added to the constructor to correct this.